CMakeLists.txt

```diff
@@ -24,6 +24,9 @@ option(EMIL_BUILD_TESTS "Enable build of the tests" ${EMIL_DEFAULTOPT})
 option(EMIL_BUILD_EXAMPLES "Enable build of the examples" ${EMIL_DEFAULTOPT})
 cmake_dependent_option(EMIL_ENABLE_COVERAGE "Enable compiler flags for code coverage measurements" Off "EMIL_BUILD_TESTS" Off)
 
+option(EMIL_BUILD_PROTOBUF "Enable build of protobuf library" On)
+cmake_dependent_option(EMIL_BUILD_UPGRADE "Enable build of upgrade library" On "EMIL_BUILD_PROTOBUF" Off)
```

Comment:
I would rather see upgrade.pack_keys being disabled. In that way the upgrade component could still be built and used. The consumer would only need to supply it's own key material.